music21/mei/base.py

```diff
@@ -1751,20 +1751,14 @@ def staffDefFromElement(elem, slurBundle=None):  # pylint: disable=unused-argume
     if post is not None:
         post = {'instrument': instrDefFromElement(post)}
     else:
-        try:
-            post = {'instrument': instrument.fromString(elem.get('label', ''))}
```

Comment:
I'm not entirely sure this is correct. If `@label` contains a valid instrument name it should still return the valid instrument, and only fall back to "UnspecifiedInstrument" if it can't find it.